### DISCUSSION TITLE
Why are some results endlessly shown as calculated?

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#22046] [Q&A] [answered]
These are just some scores set up with a flashlight or double time.
https://lazer.ppy.sh/users/13354883
![image](https://user-images.githubusercontent.com/59089206/211016261-78f087f9-abe8-4de2-b969-f74542c601bb.png)

**Comment by peppy** [comment, marked as answer]
This is because difficulty adjust is applied. These scores do not get PP until we decide how to handle them. The difficulty adjust mod icon is not visible on osu-web yet, but you can hover the empty space where mods should show to confirm this.

osu-web will need support for this eventually.
